### DISCUSSION TITLE
relationship one-to-many

### DISCUSSION THREAD

**Discussion by matv864** [igorbenav/fastcrud#86] [Q&A] [answered]
Hello, I need to create a one-to-many relationship. I searched info in documentation, but I found only many-to-many. 

My project: I have tables "card" and "articles". I want to get a card with a list of articles, 
card.id = article.card_id
There is foreign key to the card.id in the Article table. 
But I didn't found what to do in this case.

get_join, how I understand, can give only 1 article, but I need many articles.

![image](https://github.com/igorbenav/fastcrud/assets/91239322/f2a0c64f-d710-478d-903f-d004efe5549a)

**Comment by BeaverNotACat** [comment, marked as answer]
Hello. As I understand, you need to use `get_multi_joined`
[Here is the docs page](https://igorbenav.github.io/fastcrud/usage/crud/)

**Comment by igorbenav** [comment]
If you use `get_joined` or `get_multi_joined` you should be able to get the result you want. Simple usage:

```python
from fastcrud import FastCRUD

card_crud = FastCRUD(Card)
card_with_articles = await card_crud.get_joined(
    db=db,
    join_model=Article,
    join_on=Article.card_id == Card.id,
    join_type="left",
    join_prefix="article_",
)
```

Or to get multiple results:

```python
from fastcrud import FastCRUD

card_crud = FastCRUD(Card)
card_with_articles = await card_crud.get_multi_joined(
    db=db,
    join_model=Article,
    join_on=Article.card_id == Card.id,
    join_type="left",
    join_prefix="article_",
)
```
